src/locales/de.yaml

```
@@ -178,6 +183,13 @@ app:
       view: Ansicht
       reset_stats: Statistik zurücksetzen
       snooze: Schlummern
+      abort: Abbrechen
+      accept: Akzeptieren
+      adjusted: Eingestellt
```

LGTM so far, but maybe "Angepasst" would be more fitting here?
```suggestion
      adjusted: Angepasst
```